controllers/discovery/searchUser.js

```diff
@@ -75,6 +83,7 @@ const SearchUser = async (req, res) => {
       WHERE 
           (u.username ILIKE :likeQuery
           ${where_anon_dm}
+          ${filterBlockedUser}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The inclusion of the `filterBlockedUser` in the SQL query ensures that blocked users are excluded from the search results. This is a good approach to enhance user privacy.
```diff
+ ${filterBlockedUser}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->